libs/sdk-backend-tiger/src/backend/index.ts

```diff
@@ -300,7 +300,10 @@ export class TigerBackend implements IAnalyticalBackend {
         };
     };
 
-    private triggerAuthentication = (reset = false): Promise<IAuthenticatedPrincipal> => {
+    private triggerAuthentication = (
+        reset = false,
+        errorConverter: ErrorConverter = convertApiError,
```

Comment:
What is the use for this parameter? I don't see it used anywhere (passed a value).

Reply:
actually, looking through this i realized i made the fix unnecessary dangerous. will prepare rework shortly

Reply:
fix reworked